src/Pages/form/Components/StrokePredictionForm.tsx

```diff
@@ -42,33 +84,87 @@ const StrokePredictionForm: React.FC = () => {
-              name="EKG12Leads"
-              onChange={setFieldValue}
-            />
-            <SectionTitle title="NIHSS" />
-            <NIHSSSection
-              values={values.NIHSS}
-              fieldName="NIHSS"
-              onChange={setFieldValue}
-            />
-            <Button disabled={isSubmitting} type="submit">
-              Submit{""}
-            </Button>
+            <ScrollingProvider>
+              <Grid container>
+                <Grid item>
+                  <Grid container spacing={3}>
+                    <Grid item>
+                      {/* PatientInformation */}
+                      <Section id="PatientInformation">
+                        <SectionTitle title="Patient Information" />
+                        <PatientInformationSection />
+                      </Section>
+                      {/* ChiefComplaint */}
+                      <Section id="ChiefComplaint">
+                        <SectionTitle title="Chief Complaint" />
+                        <ChiefComplaintSection
+                          value={values.ChiefComplaint}
+                          name="ChiefComplaint"
+                          onChange={setFieldValue}
+                        />
+                      </Section>
+                      {/* UnderLyingDisease */}
+                      <Section id="UnderLyingDisease">
+                        <SectionTitle title="Underlying Disease" />
+                        <UnderLyingDiseaseSection />
+                      </Section>
+                      {/* Vital Signs */}
+                      <Section id="VitalSigns">
+                        <SectionTitle title="Vital Signs" />
+                        <VitalSignsSection />
+                      </Section>
+                      {/* EKG 12 Leads */}
+                      <Section id="EKG12Leads">
+                        <SectionTitle title="EKG 12 Leads" />
+                        <EKG12LeadsSection
+                          value={values.EKG12Leads}
+                          name="EKG12Leads"
+                          onChange={setFieldValue}
+                        />
+                      </Section>
+                      {/* NIHSS */}
+                      <Section id="NIHSS">
+                        <SectionTitle title="NIHSS" />
+                        <NIHSSSection
+                          values={values.NIHSS}
+                          fieldName="NIHSS"
+                          onChange={setFieldValue}
+                        />
+                      </Section>
+                      <Button
+                        disabled={isSubmitting}
+                        type="submit"
+                        className={classes.buttonButtom}
+                      >
+                        Submit{""}
+                      </Button>
+                    </Grid>
+                  </Grid>
+                </Grid>
+                {/* Sidebar */}
+                <Grid item>
```

Comment:
In my opinion, the grid layout is not necessary because SideBar position is fixed.